configapi/api_default.go

```diff
@@ -72,7 +74,7 @@ func GetDeviceGroupByName(c *gin.Context) {
 	if errGetOne != nil {
 		logger.DbLog.Warnln(errGetOne)
 	}
-	json.Unmarshal(mapToByte(rawDeviceGroup), &deviceGroup)
+	json.Unmarshal(configmodels.MapToByte(rawDeviceGroup), &deviceGroup)
```

Comment:
Error is  not handled here.

Reply:
This modification is out of the scope of this PR